documentation/serenity-js.org/docs/web-testing/your-first-web-scenario.mdx

```diff
@@ -338,17 +339,17 @@ To make your work easier, Serenity/JS modules provide [**dozens of lower-level i
 This will help you focus on **modelling your business workflows** and **designing your test scenarios** instead of
 wasting your time figuring out how to integrate with the given interface or trying to make several incompatible libraries work together.
 
-Serenity/JS activity-based code reuse model also means that as you evolve the vocabulary of your domain-specific tasks
-over time, writing test scenarios becomes easier as you're simply re-arrange existing tasks into new scenarios.
+Serenity/JS's activity-based code reuse model also means that as you evolve the vocabulary of your domain-specific tasks
```

Comment:
I'd avoid the `'s`, based on [Google style guide](https://developers.google.com/style/possessives#product,-feature,-and-company-names)

Reply:
Thanks, I like your idea of adding "The" :+1:

Reply:
> Now that being said, I'm not going to tell you how to write your website.

No, no, please keep it coming, and there are plenty of other articles on this website I'd love for you to have a look at 😉